tsconfig.json

```diff
@@ -1,18 +1,20 @@
 {
   "compilerOptions": {
-    "outDir": "./lib",
-    "declaration": true,
+    "noEmit": true,
```

Comment:
This should probably be removed 😅

Reply:
Sorry about that 😓 - See https://github.com/gcanti/docs-ts/pull/21/commits/f7613b99b3e683b18f8625c99dacf6b40455847a

Reply:
No problem 😅